types/bunyan-format/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove this rule from tslint and you can instead add `// tslint:disable-line no-empty-interface` at the error site so that this rule is not completely disabled for the library.